integration/kubernetes/README.md

```diff
@@ -67,11 +65,18 @@ bash helm-generate.sh multi-embedded
 
 ```
 
-For *singleMaster-hdfsJournal* the script relies on the `config.yaml` file you just created under the hood.
+For all modes the script relies on the `config.yaml` file in the target directory.
```

Comment:
do you mean current working directory instead of target directory? clarify what target directory means